jenkins/open-mpi.pr.command-runner.groovy

```diff
@@ -0,0 +1,90 @@
+// -*- groovy -*-
+//
+// Run the signed-off-by checker on a PR
+//
+//
+// WORKSPACE Layout:
+//   srcdir/               PR checking source tree
+//   ompi-scripts/         ompi-scripts master checkout
+
+def builder_label = "headnode"
+def pr_context = env.context_name
+def script_name = "ompi-scripts/" + env.script_name
+def target_srcdir = "srcdir"
+
+// Start by immediately tagging this as in progress...
+setGitHubPullRequestStatus(context: pr_context, message: 'In progress', state: 'PENDING')
+
+node(builder_label) {
+  stage('Source Checkout') {
+    try {
+      checkout_code(target_srcdir);
+    } catch (err) {
+      setGitHubPullRequestStatus(context: pr_context,
+                                 message: "Internal Accounting Error",
+                                 state: 'ERROR')
+      throw(err)
+    }
+  }
+
+  stage('Signed-off-by Check'){
```

Comment:
```suggestion
  stage('Checking git commits'){
```

I don't think this string is sent back to github, it we can probably just put a non-specific string here (vs. passing in another parameter with the stage name).